packages/runtime/container-runtime/garbageCollection.md

```diff
@@ -31,11 +41,6 @@ If you wish to disable this, set the `gcAllowed` option to `false` in `IGCRuntim
 See `IGCRuntimeOptions` in [containerRuntime.ts](./src/containerRuntime.ts) for more options to control GC behavior.
 
 ### Sweep phase
```

Comment:
I would mention something about how objects are only swept once the system is sure that they could never be referenced again by any active clients (a nod to Session Expiry but you don't have to talk about it that way)